state/protocol/convert.go

```diff
@@ -23,6 +23,18 @@ func ToEpochSetup(epoch Epoch) (*flow.EpochSetup, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not get epoch final view: %w", err)
 	}
+	dkgPhase1FinalView, err := epoch.DKGPhase1FinalView()
+	if err != nil {
+		return nil, fmt.Errorf("could not get dkg phase 1 final view: %w", err)
+	}
+	dkgPhase2FinalView, err := epoch.DKGPhase2FinalView()
+	if err != nil {
+		return nil, fmt.Errorf("could not get dkg phase 2 final view: %w", err)
+	}
+	dkgPhase3FinalView, err := epoch.DKGPhase3FinalView()
+	if err != nil {
+		return nil, fmt.Errorf("could not get dkg phase 3 final view: %w", err)
+	}
```

Comment:
These lines are a duplicate of the similar lines in another package, wondering can we make it a util function? 🤔 
https://github.com/onflow/flow-go/blob/c49e72cd25930c8fe9e3702eb59307a55b29469e/state/protocol/inmem/convert.go#L109-L120

Something like this:
```
func DKGFinalViews(epoch protocol.Epoch) (uint64, uint64, uint64, err)
```

Reply:
👍  https://github.com/onflow/flow-go/pull/851/commits/3d3a19b423f9420ed3651501f2f803a116561306